azure-pipelines.yml

```diff
@@ -84,7 +84,7 @@ stages:
       parameters:
         rust: $(channel)
         targets: ["$(TARGET)"]
-    - script: cargo test --target $(TARGET) --workspace
+    - script: cargo test --all-features --target $(TARGET) --workspace
```

Comment:
In the `cargo_check` job, can you add a `--no-default-features` and `--all-features`?

Reply:
cb19e48